src/user-event/paste.js

```diff
@@ -0,0 +1,120 @@
+import {getConfig as getDOMTestingLibraryConfig} from '../config'
+import {fireEvent} from '../events'
+
```

Comment:
Let's see if we can move some of the shared utilities from type until the utils and share them here.

Reply:
I've extracted some of the functions, let me know if you have any more suggestions for abstracting this further